src/platform/commandline.c

```diff
@@ -20,7 +20,11 @@
 #include "util/string.h"
 
 #include <fcntl.h>
+#ifdef _MSC_VER
+#include "platform/windows/getopt_msvc.h"
```

Comment:
Why did you rename the file? platform/windows isn't in the search path for this file, so it shouldn't matter that the names collide.